persistent/Database/Persist.hs

```diff
@@ -16,7 +16,7 @@ module Database.Persist
 -- This syntax allows you to customize the resulting Haskell datatypes and
 -- database schema. See "Database.Persist.Quasi" for details on that definition
 -- language.
---
+
```

Comment:
This was needed to properly create a heading for `Reference Schema & Dataset` [here](https://hackage.haskell.org/package/persistent-2.13.1.2/docs/Database-Persist.html)